### ISSUE TITLE
Title parsing error in custom function

### ISSUE THREAD

**Issue by noodlez04** [spencermountain/dumpster-dive#73]
Hey,
I'm running a JS script containing the following code:
```
const dumpster = require('dumpster-dive');

options = {
	file: process.argv[2],
	db: 'tinywiki',
	skip_redirects: false,
	skip_disambig: false,
	batch_size: 1000,
	custom: function(doc) {
	        console.log(doc.title());
        	return {};
	}
};
dumpster(options, () => console.log('Parsing is Done!'));
```

where I pass the script the path to the tinywiki XML file through `argv[2]` which is `./tests/tinywiki-latest-pages-articles.xml`.
The only titles being printed are `Toronto`, `Royal Cinema`, `Belleville`, while all the others print `undefined` - 3 out of 8 pages (even the duplicate pages should print at this point), however the DB is populated with the correct `title`s and `pageID`s.

BUT! If I run the same code without a custom function, it populates the DB correctly, i.e with the correct titles and everything (text, etc.), which leads me to suspect that there is something wrong passed on with the `doc` object into the custom function, but the code in:
https://github.com/spencermountain/dumpster-dive/blob/c11587c80fe1da9ff69f987bce6d7cdfd6e3b801/src/worker/02-parseWiki.js#L43

Just adds the title afterwards and pushes into the DB, making it look as if the `title` is accessible, when in fact it ISN'T accessible from within the custom function.

**Comment by spencermountain** [comment]
ah, yes this is a very good point.

hmmm. Yeah, the wtf_wikipedia parser will try to parse the title from the first-sentence. But dumpster has the proper title from it's xml, like you found.

it should slip it into the Doc object somehow. We should have a setter function for the title in wtf_wikipedia.

Will add it in.
good find!
thanks

**Comment by spencermountain** [comment]
hey, think I've fixed this in `5.1.1`, can you check it out?

**Comment by noodlez04** [comment]
Works fine now, thanks for the prompt fix!
